### PR TITLE
Document offline signing

### PR DESCRIPTION
**Pull request by crocket**
It is based on my direct experiences with offline signing.

### CODE REVIEW

**Comment by qertoip** [comment]
Thanks so much for your contribution! I will try to review it ASAP.

File: docs/interacting/offline-signing.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+```
+monero-wallet-cli --generate-new-wallet wallet-file --restore-deterministic-wallet
+```
+
+on an offline machine.
+
+When you restore from seed, you can enter seed offset passphrase. Seed and seed
+offset passphrase combine to create a new seed. You can store seed and seed
+offset passphrase in separate places so that a thief can't steal your fund
+without setaling seed and seed offset passphrase. I recommend 6 to 8 (english)
```
Comment (serhack) [inline docs/interacting/offline-signing.md]: typo: stealing

Comment (crocket) [reply docs/interacting/offline-signing.md]: Okay.

**Comment by crocket** [comment]
If you are busy, can you merge it now and modify it later?

**Comment by qertoip** [comment]
The article is now published.

Thanks again for your contribution and sorry it took this long!

The topic is tricky though and I had to add a warning on my assessment of this cold storage method.

**Pull request merged**